internal/backends/collection.go

```diff
@@ -132,6 +134,8 @@ type DeleteResult struct {
 }
 
 // Delete deletes documents in collection.
+//
+// If request database does not exist it returns 0 deleted documents.
```

Comment:
What about the collection?

Reply:
The same for the collection. Updated the comment.